src/views/LimitedRestorationExtension.vue

```diff
@@ -311,24 +302,28 @@ export default class LimitedRestorationExtension extends Vue {
     return this.getResource.userEmailOptional
   }
 
-  /** build applicant orgPerson and assign id (uuid) */
-  private parseApplicantOrgPerson (applicant: OrgPersonIF): OrgPersonIF[] {
-    const applicantOrgPerson: Array<OrgPersonIF> = []
-    applicantOrgPerson.push({
-      deliveryAddress: applicant.deliveryAddress,
-      mailingAddress: applicant.mailingAddress,
-      officer: {
-        email: applicant.officer.email,
-        firstName: applicant.officer.firstName,
-        lastName: applicant.officer.lastName,
-        middleName: applicant.officer.middleName,
-        organizationName: applicant.officer.organizationName,
-        partyType: applicant.officer.partyType,
-        id: uuidv4()
```

Comment:
The bug was the a few properties weren't being copied over here.

Reply:
This is why you were getting a null value? Since the property isn't here so it's technically not being set to anything right?

Reply:
Yes. Actually it was undefined (not null).